index.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module 'click-should-be-intercepted-for-navigation' {
+    declare function eventIsModifiedByKeyPress(event: Event): boolean;
+    declare function isLeftClick(event: Event): boolean;
+
+    function shouldInterceptEvent(event: Event): boolean;
```

Comment:
hmm, I wonder if these should be `MouseEvent` instead of `Event`?

Reply:
Up to you

Reply:
If it would be more correct to expect `MouseEvent`, then that's what I'd like 😅

Reply:
`Event`s are tough without knowing every way it's utilized. I would be a little concerned being too strict if it's going to be attached to a component that types its output as `Event`

Reply:
Fine by me 👍